content/community/contributing-to-documentation.md

```diff
@@ -25,58 +26,67 @@ The more general contribution process is described in the [Cucumber Community Co
+If you have any questions about these issues, you can add comment on the issue itself or contact us on Slack.
 
 ## General writing style
+In general, the documentation should be brief and to the point.
 
+> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away - Antoine de Saint Exupéry
+
+Some guidelines:
 * Every page should start with an informational/motivational paragraph
 * Paragraphs should be short enough to be readable, but long enough to develop an idea
-* Every page should start with a `h1` heading. Sections use `h2`. Subsections use `h3`
+* Every page should start with a `h1` heading. Sections use `h2`; subsections use `h3`
 * Break long lines. Insert a new line at around column 80. This is important because review comments can only be added to a line.
 * Write in present tense
-* Use unemotional language, but try to make it a little entertaining (this is hard!)
-* Write in a platform-neutral way as much as possible
-  * Cucumber is implemented in several languages, and the docs should not assume a particular platform
-  * Some good examples of cross-platform/language docs are [Stripe](https://stripe.com/docs/api) and [.NET](https://msdn.microsoft.com/en-us/library/system.array(v=vs.110).aspx)
-* Use [code blocks](#language-specific-source-code-and-paragraphs) for all code examples (except Gherkin)
-* Use [language blocks](#Language-specific-text-fragments) for text that is only relevant for (a) specific language(s)
-  * If you're only familiar with one programming language, just add an example for that language; someone
-    else will fill the gaps for the other languages!
-  * You can ask for help with the other languages in the help channel for that language on Slack, or in your GitHub pull request / issue
-* All documents should use [British English](https://en.wikipedia.org/wiki/British_English)
-  * Contributions in [American English](https://en.wikipedia.org/wiki/American_English) is fine - editors will do the translation.
+* Use neutral language, but try to make it a little entertaining (this is hard!)
+* Write in a platform-neutral way as much as possible. Cucumber is implemented in several languages, and the docs should not assume a particular platform
+* Use [code blocks](#language-specific-source-code-and-paragraphs) for all code examples (except Gherkin) and paragraphs related to one or more specific languages
+* Use [language blocks](#Language-specific-text-fragments) for text that is only relevant for one or more specific languages
+* Mark [Polyglot pages](#polyglot-pages) as needed
+* All documents should use [British English](https://en.wikipedia.org/wiki/British_English). Contributions in [American English](https://en.wikipedia.org/wiki/American_English) is fine - editors will do the translation.
```

Comment:
Should be "Contributions in American English are fine" rather than "is fine".

Reply:
```suggestion
* All documents should use [British English](https://en.wikipedia.org/wiki/British_English). Contributions in [American English](https://en.wikipedia.org/wiki/American_English) are fine - editors will do the translation.
```